pay-api/src/pay_api/services/base_payment_system.py

```diff
@@ -150,7 +151,8 @@ def ensure_no_payment_blockers(self, payment_account: PaymentAccount) -> None:
             # Note NSF (Account Unlocking) is paid using DIRECT_PAY - CC flow, not PAD.
             current_app.logger.warning(f'Account {payment_account.id} is frozen, rejecting invoice creation')
             raise BusinessException(Error.PAD_CURRENTLY_NSF)
-        if Invoice.has_overdue_invoices(payment_account.id):
+        if flags.is_on('enable-eft-payment-method', default=False) and \
+            Invoice.has_overdue_invoices(payment_account.id):
```

Comment:
I think the second check could cripple the database, it's feature flagged though